test/validators/movie.test.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const Joi = require('joi');
+
+const MovieValidator = require('../../lib/validators/movie');
+
+describe('movie validator', () => {
+
+  it('succeeds when all allowed params are included', () => {
+    const title = 'title';
+    const release_year = 2017;
+    const payload = { title, release_year };
```

Comment:
While this is fine, I don't think it's necessary to extract all the values of the payload into their own variables. If you ever need to access them you can simply do `payload.title`. Personally I think it's easier to decode what each value is as the key and value are next to each other rather than having to go back and look what the value of the variable was. Specially if there are tons of keys such as our API serializers.

```js
const title = 'title';
const release_year = 2017;
const payload = { title, release_year };

// vs.

const payload = {
  title: 'title',
  release_year: 2017
};
```